apps/web/components/CardWithFeaturedItems/CardWithFeaturedItems.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  FocusableBox,
+  Hidden,
+  Link,
+  Tag,
+  Text,
+} from '@island.is/island-ui/core'
+import * as styles from './CardWithFeaturedItems.css'
+import { LinkType, useLinkResolver } from '@island.is/web/hooks'
+import { Featured } from '@island.is/web/graphql/schema'
+
+type CardWithFeaturedItemsProps = {
+  heading: string
+  imgSrc: string
+  alt: string
```

Comment:
I wonder if setting the imgAlt prop to be optional and defaulting it to an empty string would be preferred since in most cases the image is decorative.

The alt text in this example here gives no further context so I'd suggest setting the alt text to be an empty string
![image](https://github.com/island-is/island.is/assets/43557895/8610e298-a573-4473-a91c-7f1eb3745c2a)